src/ai/index.js

```diff
@@ -239,6 +239,7 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
         adjustTextAreaSize();
         setupEventListeners();
         setupSystemEventListeners();
+        hideEndpointApiKeyAndShowSearchTextArea();
```

Comment:
Calling this immediately so that it skips the ApiKey step. Should probably just remove that code entirely.